airflow/utils/decorators.py

```diff
@@ -18,54 +18,55 @@
 from __future__ import annotations
 
 import sys
-from collections import deque
 from typing import Callable, TypeVar
 
+import libcst as cst
+
 T = TypeVar("T", bound=Callable)
 
 
+class _TaskDecoratorRemover(cst.CSTTransformer):
+    def __init__(self, task_decorator_name):
+        self.decorators_to_remove = {
+            "setup",
+            "teardown",
+            "task.skip_if",
+            "task.run_if",
+            task_decorator_name.strip("@"),
+        }
+
+    def _is_task_decorator(self, decorator: cst.Decorator) -> bool:
```

Comment:
```suggestion
    def _is_task_decorator(self, decorator_node: cst.Decorator) -> bool:
        decorator_expression = decorator_node.decorator
```

I think we can make it more readable here like this

Reply:
@josix this one is missed